contracts/cw20-ics20/src/ibc.rs

```diff
@@ -170,8 +178,9 @@ pub fn ibc_packet_receive(
             attributes: vec![
                 attr("action", "receive"),
                 attr("success", "false"),
-                attr("error", err),
+                attr("error", err.to_string()),
```

Comment:
I know you cannot implement attr for both `impl Into<String>` and `impl ToString` as some structs implement both and you get a compiler error there.

Is there a way to say `K: ToString & !Into<String>` ? I think I looked into this some months ago and found it worked in unstable rust (and used to optimise std lib Vec for example)

Reply:
Ayep. It's probably just something we have to live with.